lib/bot.js

```diff
@@ -152,6 +153,11 @@ class Bot {
       return;
     }
 
+    if (message.user === 'USLACKBOT' && this.muteSlackbot) {
```

Comment:
Tiny detail, but maybe swap the checks? Since `false` is default, i.e. `if (this.muteSlackbot && ...`